tests/acceptance/test_multi_threading.py

```diff
@@ -218,7 +218,7 @@ def test___shared_interpreter___unregister_events_during_other_acquisitions_with
             )
             for i, device in enumerate(multi_threading_test_devices)
         ]
-        samples_per_chan = [1000 for _ in tasks]
+        samples_per_chan = [(100 * (i + 8)) for i in range(len(tasks))]
```

Comment:
I don't think it's necessary to add the new test coverage to the unregister-during-other-acquisitions test.